PerimeterXModule/Internals/TemplateFactory.cs

```diff
@@ -54,6 +54,16 @@ private static string getTemplateString(string template)
 			props.Add("hostUrl", string.Format(pxConfiguration.CollectorUrl, pxConfiguration.AppId));
 			props.Add("captchaType", pxConfiguration.CaptchaProvider);
 
+			if (pxConfiguration.FirstPartyEnabled)
+			{
+				props.Add("jsClientSrc", string.Format("/{0}/init.js", pxConfiguration.AppId.Substring(2)));
+				props.Add("firstPartyEnabled", "1");
+
+			}
+			else
+			{
+				props.Add("jsClientSrc", string.Format("//client.perimeterx.net/{0}/main.min.js", pxConfiguration.AppId));
```

Comment:
Let's have a constant for `client.perimeterx.net`. 
We can even have it as a config. On NGINX for example, having it as a config was quite useful when we were testing first party with pxClientServer (at some point during testing with Guy we sent the requests to `client-stg.perimeterx.net`).

Reply:
You actually do have it already - `ClientHostUrl`.

Reply:
Also, let's have `main.min.js` as a constant (not config) as well, and use in all corresponding places.

Reply:
👍